translator/kapua/kura/src/main/java/org/eclipse/kapua/translator/kura/kapua/TranslatorLifeAppsKuraKapua.java

```diff
@@ -50,6 +50,10 @@ public KapuaAppsMessage translate(KuraAppsMessage kuraAppsMessage)
         AccountService accountService = locator.getService(AccountService.class);
         Account account = accountService.findByName(kuraAppsMessage.getChannel().getScope());
 
+        if (account == null) {
+            throw new KapuaEntityNotFoundException(Account.class.toString(), kuraAppsMessage.getChannel().getScope());
```

Comment:
Hi @riccardomodanese ,

please change the throw parameter from `Account.class.toString()` to `Account.getType()`
Same for all other changes.

Regards,

_Alberto_